packages/dapi/scripts/api.js

```diff
@@ -107,17 +107,16 @@ async function main() {
     tenderdashLogger.warn(`Connection closed: ${error.code}`);
   });
 
-  tenderDashWsClient.on('disconnect', () => {
-    tenderdashLogger.fatal('Disconnected from Tenderdash... exiting');
+  const blockchainListener = new BlockchainListener(tenderDashWsClient);
+
+  tenderDashWsClient.connect().then(() => {
+    blockchainListener.start();
+  }).catch((e) => {
+    tenderdashLogger.fatal(`Not able to connect to tenderdash: ${e}`);
```

Comment:
Double-check if `process.exit(1)` below is correct, if I understand correctly, we intended to not exit in this case.

Reply:
Yes, it will happen it case of the bug in connect function or maRetires != -1 (which is no the case)